2017-08-30 09:35:21,094 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 09:35:21,145 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 09:35:21,147 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,164 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,164 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,164 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,165 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,165 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,165 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,165 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,166 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,166 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,166 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,166 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,166 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,167 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,167 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,167 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,167 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,167 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,167 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,168 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,168 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,168 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,168 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,168 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,168 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,169 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,169 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,169 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,169 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,169 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:21,169 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:35:51,173 [Druid-ConnectionPool-Create-5810563] ERROR [com.alibaba.druid.pool.DruidDataSource] - create connection error, url: 
java.sql.SQLException: connect error, url , driverClass sun.jdbc.odbc.JdbcOdbcDriver
	at com.alibaba.druid.pool.DruidAbstractDataSource.createPhysicalConnection(DruidAbstractDataSource.java:1434)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1861)
2017-08-30 09:53:51,239 [main] INFO  [application.dao.DbUtil] - 驱动加载完成
2017-08-30 09:54:38,873 [main] INFO  [application.dao.DbUtil] - 驱动加载完成
2017-08-30 09:55:03,556 [main] INFO  [application.dao.DbUtil] - 驱动加载完成
2017-08-30 09:56:16,658 [main] INFO  [application.dao.DbUtil] - 驱动加载完成
2017-08-30 09:59:05,717 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 09:59:05,747 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 09:59:06,434 [Druid-ConnectionPool-Create-21417132] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 10:02:42,129 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 10:02:42,150 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 10:02:42,152 [main] INFO  [application.dao.DbUtil] - 数据源加载成功,driver:sun.jdbc.odbc.JdbcOdbcDriver,url:jdbc:odbc:IDataBase,username:dba
2017-08-30 10:02:42,741 [Druid-ConnectionPool-Create-30031664] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 10:03:46,012 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 10:03:46,033 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 10:03:46,035 [main] INFO  [application.dao.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 10:03:47,120 [Druid-ConnectionPool-Create-14747609] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:28:54,727 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:28:54,795 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:28:54,797 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 11:28:54,798 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:28:54,801 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:28:54,833 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:28:54,884 [Druid-ConnectionPool-Create-1222657] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:28:54,920 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:29:39,984 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:29:40,010 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:29:40,013 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 11:29:40,013 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:29:40,021 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:29:40,027 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:29:40,062 [Druid-ConnectionPool-Create-6989294] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:29:40,102 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:30:45,821 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:30:45,861 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:30:45,864 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 11:30:45,864 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:30:45,876 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:30:45,895 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:33:25,176 [Druid-ConnectionPool-Create-28568594] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:34:37,921 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:35:36,040 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:35:36,068 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:35:36,070 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 11:35:36,073 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:35:36,090 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:35:36,096 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:35:36,873 [Druid-ConnectionPool-Create-11456467] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:35:36,950 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:36:53,725 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:36:53,747 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:36:53,749 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 11:36:53,749 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:36:53,752 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:36:53,757 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:36:54,309 [Druid-ConnectionPool-Create-13514081] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:36:54,397 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:37:48,101 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:37:48,122 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:37:48,124 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 11:37:48,174 [Druid-ConnectionPool-Create-7936257] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:39:01,816 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:39:02,005 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:39:02,043 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:39:02,045 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:39:02,045 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:39:02,048 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:39:02,054 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:39:02,610 [Druid-ConnectionPool-Create-13514081] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:39:02,666 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:39:53,063 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:39:58,303 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:39:58,647 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:39:58,664 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:39:58,666 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:40:00,201 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:40:39,998 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:41:03,313 [Druid-ConnectionPool-Create-31642721] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:41:03,652 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:42:45,101 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:42:45,237 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:42:45,256 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:42:45,258 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:42:45,258 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:42:45,260 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:42:45,266 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:42:45,841 [Druid-ConnectionPool-Create-3686138] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:42:45,915 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:46:06,339 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:46:06,596 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:46:06,632 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:46:06,636 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:46:06,636 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:46:06,639 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:46:06,643 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:46:07,414 [Druid-ConnectionPool-Create-8931300] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:46:07,520 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:46:58,872 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:46:59,022 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:46:59,049 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:46:59,051 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:46:59,052 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:46:59,054 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:46:59,060 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:47:02,674 [Druid-ConnectionPool-Create-27045597] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:48:14,823 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:48:30,850 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:48:31,023 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:48:31,054 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:48:31,056 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:48:31,056 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:48:31,063 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:48:31,069 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:48:31,105 [Druid-ConnectionPool-Create-13898005] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:48:31,167 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:49:46,062 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:49:46,221 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:49:46,257 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:49:46,259 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:49:46,260 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:49:46,262 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:49:46,269 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:49:57,767 [Druid-ConnectionPool-Create-24136676] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:50:00,176 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:50:21,192 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:50:21,400 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:50:21,430 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:50:21,432 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:50:21,432 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:50:21,435 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:50:21,441 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:50:26,599 [Druid-ConnectionPool-Create-25586585] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:50:58,145 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:52:14,029 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:52:14,233 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:52:14,257 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:52:14,259 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:52:14,259 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:52:14,262 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:52:14,268 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:52:16,397 [Druid-ConnectionPool-Create-149539] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:52:18,046 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:53:03,380 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:53:03,518 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:53:03,547 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:53:03,553 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:53:03,553 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:53:03,556 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:53:03,562 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:53:04,142 [Druid-ConnectionPool-Create-27045597] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:53:04,218 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:53:49,663 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:53:49,802 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:53:49,833 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:53:49,836 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:53:49,836 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:53:49,839 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:53:49,845 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:53:49,879 [Druid-ConnectionPool-Create-27025461] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:53:49,921 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 11:57:18,426 [main] INFO  [application.util.AppContext] - 配置项加载完成！
2017-08-30 11:57:18,582 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 11:57:18,608 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 11:57:18,611 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: dba
2017-08-30 11:57:18,611 [main] INFO  [application.cache.App] - regist tableDao
2017-08-30 11:57:18,615 [main] INFO  [application.cache.App] - regist tableService
2017-08-30 11:57:18,620 [main] INFO  [application.cache.App] - cache加载完成
2017-08-30 11:57:19,144 [Druid-ConnectionPool-Create-24136676] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 11:57:19,200 [main] INFO  [application.dao.TableDao] - search success! sql ---> select t.db_id dbId,t.dbeng enName,t.dbchin chName,t.dbtype dbType,t.vol_dbid volDbId from jktable t
2017-08-30 12:00:53,001 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 12:00:53,020 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 12:00:53,022 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 12:00:53,510 [Druid-ConnectionPool-Create-5810563] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 12:01:31,949 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 12:01:31,968 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 12:01:31,970 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 12:01:32,585 [Druid-ConnectionPool-Create-14747609] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 12:01:44,292 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 12:01:44,312 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 12:01:44,314 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 12:01:44,345 [Druid-ConnectionPool-Create-5810563] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
2017-08-30 12:01:58,533 [main] ERROR [com.alibaba.druid.pool.DruidDataSource] - testWhileIdle is true, validationQuery not set
2017-08-30 12:01:58,553 [main] INFO  [com.alibaba.druid.pool.DruidDataSource] - {dataSource-1} inited
2017-08-30 12:01:58,555 [main] INFO  [application.dao.util.DbUtil] - 数据源加载成功,driver: sun.jdbc.odbc.JdbcOdbcDriver ,url: jdbc:odbc:IDataBase ,username: 
2017-08-30 12:01:58,585 [Druid-ConnectionPool-Create-5810563] WARN  [com.alibaba.druid.pool.DruidConnectionHolder] - getHoldability unsupported
java.lang.UnsupportedOperationException
	at sun.jdbc.odbc.JdbcOdbcConnection.getHoldability(Unknown Source)
	at com.alibaba.druid.pool.DruidConnectionHolder.<init>(DruidConnectionHolder.java:86)
	at com.alibaba.druid.pool.DruidDataSource.put(DruidDataSource.java:1721)
	at com.alibaba.druid.pool.DruidDataSource$CreateConnectionThread.run(DruidDataSource.java:1890)
